.editorconfig

```diff
@@ -67,6 +67,7 @@ dotnet_analyzer_diagnostic.category-Style.severity = warning
 dotnet_diagnostic.IDE0005.severity = warning    # Using directive is unnecessary.
 dotnet_diagnostic.CA1848.severity = none        # Do not encourage LoggerMessage delegates in every instance
 dotnet_diagnostic.CA2254.severity = suggestion  # Logging template should not vary between calls
+dotnet_diagnostic.IDE0058.severity = suggestion
```

Comment:
Do we have tons of expressions that we ignore the result for?

Reply:
yes, probably hundreds 😞  
All the string builders but others as well...

Reply:
Ok, I guess it's ok for now.
Add a comment though?